### ISSUE TITLE
CryptoError: Error(

### ISSUE THREAD

**Issue by pocmo** [mozilla-mobile/android-components#5677]
https://sentry.prod.mozaws.net/operations/fenix-fennec-nightly/issues/7131337/

```
CryptoError: Error(

Crypto error: Error(

Crypto error: Decryption error))
    at mozilla.appservices.push.RustError.intoException(RustError.kt:14)
    at mozilla.appservices.push.PushManager.decrypt(PushManager.kt:16)
    at mozilla.components.feature.push.RustPushConnection.decrypt(Connection.kt:3)
    at mozilla.components.feature.push.AutoPushFeature$onMessageReceived$1.invokeSuspend(AutoPushFeature.kt:15)
    at mozilla.components.feature.push.AutoPushFeature$onMessageReceived$1.invoke
...
(6 additional frame(s) were not displayed)

Error(

Crypto error: Error(

Crypto error: Decryption error))
```

**Comment by jonalmeida** [comment]
Adding some notes here:

> TL;DR - we don't need to crash, but we should log them if they count is high.
> 
> A client can send us a valid message but without encrypting the message correctly or at all. We can't know until we try to decrypt it, so logging the amount of times this happens will give us some into a bad crypto implementation in the wild although not the library being used.
>

**Comment by LaurentiuApahideanSV** [comment]
@pocmo Hi! Can you please provide steps to reproduce this? Thanks!

**Comment by ebalazs-sv** [comment]
Since this is an issue that needs Sentry access, st3fan confirmed that we can move this to Done. Thanks!
